manifests/charts/base/templates/zzz_profile.yaml

```diff
@@ -15,6 +15,7 @@ However, we can workaround this by placing all of (1) under a specific key (.Val
 We can then merge the profile onto the defaults, then the user settings onto that.
 Finally, we can set all of that under .Values so the chart behaves without awareness.
 */}}
+{{- $globals := pick ($.Values.global | default dict) "hub" "tag" "variant" "pollPolicy" }}
```

Comment:
This wont' work in all cases, because not all charts template the same way (some use e.g. `cni.hub`, some use `hub`, some use `imagePullPolicy`, some use pullPolicy`

So you (unfortunately) cannot (correctly) do this collapse and have it work properly for all charts, unless you have

- every chart do its own slightly unique image templating (leads to inconsistency, as we have seen)
- descope the "problem charts" to force consistency 

https://github.com/istio/istio/pull/51517 handles both of the latter for this reason.

Reply:
To restate this:

You can collapse the globals to the defaults, but not all charts use the same keys for `hub`/`tag`/etc - so that's no guarantee they would actually respect the collapse values.

You can fix that by duplicating a mostly-identical template in each chart with local tweaks for the locally-divergent values paths, or by doing what #51517 does (more robust descoping of the problem values for the problem charts + single consistent image template).

The solution that doesn't require copypasting slightly different templates in each chart is more resistant to drift (well, the profiles ARE copypasted to each chart, but it's automated, and identical for each, which is better).